CHANGELOG.md

```diff
@@ -15,5 +20,6 @@ Added supporting files and documentation for the custom buildpack use case
 
 The first tagged version.
 
-[Unreleased]: https://github.com/conjurinc/cloudfoundry-conjur-buildpack/compare/v0.2.0...HEAD
+[Unreleased]: https://github.com/conjurinc/cloudfoundry-conjur-buildpack/compare/v0.2.1...HEAD
 [0.2.0]: https://github.com/conjurinc/cloudfoundry-conjur-buildpack/compare/v0.1.0...v0.2.0
+[0.2.1]: https://github.com/conjurinc/cloudfoundry-conjur-buildpack/compare/v0.2.0...v0.2.1
```

Comment:
I think this line should also appear below Unreleased. The newer releases are actually added to the top.